test/plots/piePlotTests.ts

```diff
@@ -165,6 +165,31 @@ describe("Plots", () => {
         svg.remove();
       });
 
+      it("entityAt with defaults", () => {
```

Comment:
"`entitiesAt()`". Also what are the defaults? A more descriptive name for the test would be better.